backend/Tasque.Core.Common/Entities/Task/DropdownField.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasque.Core.Common.Entities.Task
+{
+    public class DropdownField
+    {
+        public string[] Fields { get; set; }
```

Comment:
Not for this PR, but can we make all fields more generic?

Reply:
Removed from this PR. Yes and I guess I know how, but I need to use this data in branch with cosmosDb functionality